Server/src/main/java/gotcha/server/Domain/RatingModule/UserRateCalculator.java

```diff
@@ -64,33 +109,92 @@ public void update_user_rating(Rider rider, Ride ride, int number_of_rides){
     @Override
     public void update_tables(Dictionary<String, Dictionary<Integer, Integer>> tables){
 
-        this.speed_changes_per_minute = tables.get("speed_changes_per_minute");
-        this.brakes_per_minute = tables.get("brakes_per_minute");
-        this.sharp_turns_per_minute = tables.get("sharp_turns_per_minute");
-        this.alerts_per_minute = tables.get("alerts_per_minute");
-        this.on_sidewalk = tables.get("on_sidewalk");
+        this.speed_changes_per_minuteTable = tables.get("speed_changes_per_minute");
+        this.brakes_per_minuteTable = tables.get("brakes_per_minute");
+        this.sharp_turns_per_minuteTable = tables.get("sharp_turns_per_minute");
+        this.alerts_per_minuteTable = tables.get("alerts_per_minute");
+        this.on_sidewalkTable = tables.get("on_sidewalk");
     }
 
 
     @Override
     public void set_tables() {
```

Comment:
Maybe we should Use another configuration class for the variables instead of going over the text file line by line.
Also I'm not sure we will ever reach the lines that update the tables since we use continue inside each if that sets the variables.

Reply:
It's tested and work fine